src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("core.basesyntax.Bulldozer started its work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("core.basesyntax.Bulldozer stopped its work");

```

Comment:
```suggestion
        System.out.println("Bulldozer started its work");
    }

    @Override
    public void stopWork() {
        System.out.println("Bulldozer stopped its work");
```
remove package name from each method